historianutils/historianutils.go

```diff
@@ -31,6 +31,9 @@ var (
 	// ServiceDumpRE is a regular expression to match the beginning of a service dump.
 	ServiceDumpRE = regexp.MustCompile(`^DUMP\s+OF\s+SERVICE\s+(?P<service>\S+):`)
 
+	// ServiceDumpWithTagRE is a regular expression to match the beginning of a service dump with tag.
+	ServiceDumpWithTagRE = regexp.MustCompile(`^DUMP\s+OF\s+SERVICE\s+(?P<serviceTag>\S+)\s+(?P<service>\S+):`)
```

Comment:
Is the service tag useful? If not is it possible to created just a single regular expression that works with or without service tag?